templates/tab/js/dashboard/README.md

```diff
@@ -314,16 +314,14 @@ try {
 } catch (e) {}
 ```
 
-### From the back-end(use application permissions)
+### Call graph api from the back-end(use application permissions)
 
-If you want to call a Graph API from the back-end, you can refer to the following steps. In this tutorial, we use `sendActivityNotification` API for example.
+If you want to call a Graph API from the back-end, you can refer to the following steps.
 
 1. [Step 1: Consent application permissions first](#step-1-consent-application-permissions-first)
 2. [Step 2: Add an Azure Function](#step-2-add-an-azure-function)
-3. [Step 3: Get the `installation id` of your Dashboard app](#step-3-get-the-installation-id-of-your-dashboard-app)
-4. [Step 4: Add your logic in Azure Function](#step-4-add-your-logic-in-azure-function)
-5. [Step 5: Edit manifest file](#step-5-edit-manifest-file)
-6. [Step 6: Call the Azure Function from the front-end](#step-5-call-the-azure-function-from-the-front-end)
+4. [Step 3: Add your logic in Azure Function](#step-4-add-your-logic-in-azure-function)
```

Comment:
The number of this part seems to have been forgotten to change.

Reply:
> The number of this part seems to have been forgotten to change.

Good catch. The number in preview is correct, so i didn't find this mistake. Thanks~